cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -63,17 +71,16 @@ def catalog_item(appliance, request, config_manager, dialog, catalog):
     return catalog_item
 
 
-@pytest.mark.tier(2)
-@pytest.mark.ignore_stream('upstream')
-def test_order_tower_catalog_item(appliance, catalog_item, request):
-    """Tests order catalog item
+def test_order_tower_catalog_item(appliance, catalog_item, request, job_type):
```

Comment:
Why remove the tier marker here? I think both tests should have one.

Reply:
In the `pytestmark` at the top, all tests are marked with `tier(2)`